### DISCUSSION TITLE
讓python當接歌DJ

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#737] [General]
我是一個非常喜歡音樂的人
可以說是無樂不歡
歌單裡非常多歌
但每次前後曲風差異太大讓人聽了沒有那麼爽快
於是我發現有能按bpm（音樂速度）排序的python套件
十分期待
https://zhuanlan.zhihu.com/p/265124519

**Comment by HsuChen88** [comment]
按照這個排法，原本坐著聽，聽一聽就跳起來了，旁邊的人越來越靠近電話。
